src/KafkaUtil.js

```diff
@@ -11,7 +11,9 @@ module.exports = class KafkaUtil extends events.EventEmitter {
         super()
         this.dataTopic = dataTopic
         this.partitioner = partitioner
-        this.kafkaClient = kafkaClient || new kafka.KafkaClient(zookeeper, `streamr-kafka-producer-${Date.now()}`)
+        this.kafkaClient = kafkaClient || new kafka.KafkaClient({
+            kafkaHost: zookeeper
```

Comment:
@mirpo this doesn't look right. You're passing the zookeeper address as the kafka host address. According to [the docs](https://www.npmjs.com/package/kafka-node) you should be passing one or more kafka node addresses to it.

Older versions of `kafka.KafkaClient` took the zookeeper address as the first argument. I wonder if this broke in a `kafka-node` version upgrade?